mappings/net/minecraft/client/render/debug/WorldGenAttemptDebugRenderer.mapping

```diff
@@ -1 +1,17 @@
 CLASS net/minecraft/class_873 net/minecraft/client/render/debug/WorldGenAttemptDebugRenderer
+	FIELD field_4634 client Lnet/minecraft/class_310;
+	FIELD field_4635 values Ljava/util/List;
+	FIELD field_4636 colorsGreen Ljava/util/List;
+	FIELD field_4637 colorsAlpha Ljava/util/List;
+	FIELD field_4638 colorsBlue Ljava/util/List;
+	FIELD field_4639 colorsRed Ljava/util/List;
```

Comment:
What are these? These should have better names, `colorsGreen` makes it seem like it's a list of colors which are green. Do you mean `colorGreens` (the green components of the colors).

Reply:
I'd have to check the code again, but basically each list corresponds to a rectangle drawn of their respective colours.

Reply:
> I'd have to check the code again

That confirms that the name should be more descriptive

> but basically each list corresponds to a rectangle drawn of their respective colours.

Then `greenRectangles` would be better.

Reply:
For every attempt it adds the `position`, `red`, `green`, `blue`, and `alpha` values.

```
    public void addWorldGenAttempt(BlockPos pos, float value, float red, float green, float blue, float alpha) {
        this.positions.add(pos);
        this.values.add(value);
        this.colorsAlpha.add(alpha);
        this.colorsRed.add(red);
        this.colorsGreen.add(green);
        this.colorsBlue.add(blue);
    }
```

Position determines the position of the box, value is the size of the box, and r/g/b/a the colour of the box.

```
for (int i= 0; i< this.positions.size(); ++i) {
		BlockPos pos = this.positions.get(i);
		Float value = this.values.get(i) / 2;

		WorldRenderer.buildBox(buffer,
			pos.getX() + 0.5 - value - camX,
			pos.getY() + 0.5 - value - camY,
			pos.getZ() + 0.5 - value - camZ,
			pos.getX() + 0.5 + value - camX,
			pos.getY() + 0.5 + value - camY,
			pos.getZ() + 0.5 + value - camZ,
			this.colorsRed.get(i),
			this.colorsGreen.get(i),
			this.colorsBlue.get(i),
			this.colorsAlpha.get(i)
		);
}
```

Reply:
So then it should be `colorReds` (red components of the color), not `colorsRed`.